src/core/Akka/Dispatch/ExecutorService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Akka.Actor;
+
+namespace Akka.Dispatch
+{
+    /// <summary>
+    /// INTERNAL API
+    /// 
+    /// Used by the <see cref="Dispatcher"/> to execute asynchronous invocations
+    /// </summary>
+    public abstract class ExecutorService
```

Comment:
Technically speaking I guess this doesn't need to be marked as "internal" since it can be configured via HOCON. It's possible to load an externally defined `ExecutorService` via a FQN.